src/response/generate_api_token_response.rs

```diff
@@ -0,0 +1,17 @@
+use std::time::SystemTime;
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct ApiKeyWithEndpoint {
+    pub api_key: String,
+    pub endpoint: String,
+}
```

Comment:
You should not derive Debug here - you should impl it and redact the api_key, otherwise this will show up in logs incidentally.

Reply:
fixed in https://github.com/momentohq/client-sdk-rust/pull/132/commits/a10baf2df4d874788a9eae80e49a54fbedf939bd